README.md

```diff
@@ -29,17 +29,17 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
 
 ```json
 {
-  "plugins": [
-    "@semantic-release/commit-analyzer",
-    "@semantic-release/release-notes-generator",
-    [
-      "semantic-release-slack-bot",
-      {
-        "notifyOnSuccess": false,
-        "notifyOnFail": true
-      }
-    ]
-  ]
+	"plugins": [
```

Comment:
Seems that `prettier` did this.

If you don't like it, I can re-commit without prettier :-)

Reply:
Using tabs make it format weird in github. If you change this and the other change this PR looks perfect to me 👍